package C4.T4;

public class Try1 {

    public static void main(String[] args) {

        // SIN RY - CATCH

        int[] array = new int[20];
        array[-3] = 24;

        /*

        CON TRY - CATCH
        
        try {
            int[] array = new int[20];
            array[-3] = 24; // Aquí es donde ocurre el error
          } catch (ArrayIndexOutOfBoundsException e) {
            System.out.println("¡Hey! No puedes poner un juguete en un compartimiento que no existe!");
          }

        */

    }

}
